app/build.gradle

```diff
@@ -100,6 +100,9 @@ dependencies {
     }
     annotationProcessor 'androidx.room:room-compiler:2.2.6'
     implementation 'de.hdodenhof:circleimageview:3.1.0'
+
+    // https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.0.1'
```

Comment:
What is this implementation for?

Reply:
It's a library that allows you to read/write Maps from/to JSON files. I use it in the UserDatabase class.